### ISSUE TITLE
mexico_nuevo_leon: remove base class

### ISSUE THREAD

**Issue by yolile** [open-contracting/kingfisher-collect#666]
After #665 mexico_nuevo_leon_base is no longer needed and that code can be moved to mexico_nuevo_leon_releases and add `skip_pluck`

**Comment by jpmckinney** [comment]
`skip_pluck` should be restored to one of them.
